bokeh/charts/_chart.py

```diff
@@ -248,7 +248,7 @@ def show(self):
         elif self._options.filename is False and \
                         self._options.server is False and \
                         self._options.notebook is False:
-            print("You have a provide a filename (filename='foo.html' or"
+            print("You have to a provide a filename (filename='foo.html' or"
```

Comment:
Yeah, that reads better.  Just tried to limit my change to make it correct with the smallest change.